specification/customer-insights/resource-manager/Microsoft.CustomerInsights/2017-04-26/customer-insights.json

```diff
@@ -3149,6 +3149,378 @@
+          },
+          "202": {
+            "description": "Accepted. The create Prediction operation is accepted."
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "get": {
+        "tags": [
+          "Predictions"
+        ],
+        "operationId": "Predictions_Get",
+        "x-ms-examples": {
+          "Predictions_Get": {
+            "$ref": "./examples/PredictionsGet.json"
+          }
+        },
+        "description": "Gets a Prediction in the hub.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "hubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the hub."
+          },
+          {
+            "name": "predictionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Prediction."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Successfully get the Prediction.",
+            "schema": {
+              "$ref": "#/definitions/PredictionResourceFormat"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Predictions"
+        ],
+        "operationId": "Predictions_Delete",
+        "x-ms-examples": {
+          "Predictions_Delete": {
+            "$ref": "./examples/PredictionsDelete.json"
+          }
+        },
+        "description": "Deletes a Prediction in the hub.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "hubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the hub."
+          },
+          {
+            "name": "predictionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Prediction."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The Prediction is deleted."
+          },
+          "202": {
+            "description": "Accepted. The delete operation is accepted."
+          }
```

Comment:
Took another look at our service code, it actually doesn't return 204 when not found, but returns 404.